service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -300,4 +367,66 @@ void resetInvocations() {
             .service(FPL)
             .siteId("TEST_SITE_ID");
     }
+
+    private CreditAccountPaymentRequest buildCreditAccountPaymentRequest(String caseReference,
+                                                                         String customerReference,
+                                                                         FeeDto feeDto) {
+        return CreditAccountPaymentRequest.builder()
+           .accountNumber("PBA123")
+           .caseReference(defaultIfBlank(caseReference, BLANK_PARAMETER_VALUE))
```

Comment:
I don't those two lines - you are using your service under test to get expected outcome! It's wrong! I can break the service by returning empty string always and your test is going to pass, you should have your own expectations in tests.